### DISCUSSION TITLE
UI Design

### DISCUSSION THREAD

**Discussion by davcortez** [Ecuador-In-Tech/web#23] [Ideas]
I’m wondering how to approach the UI design for the entire site. I was looking at the Figma, but the design doesn’t cover many aspects. What about responsive design? It’s worth thinking in a mobile-first way. 

@nsandoya @IvanM9

**Comment by nsandoya** [comment]
Hi @davcortez ,

I understand and share your concerns. I'd like to share with you a **redesign** I was working on early in the morning, based on our **current website** and the **volunteer's proposal** shared with the group a couple of days ago: 

- **New prototype:** https://www.figma.com/proto/x7GTNPvqJwtyzjfKX5NqH9/ECinTech?node-id=431-13687&t=sCOgDXjo3cLo8kHR-0&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1
- **Volunteer's proposal:** _You can find the link in the core group chat_

Meanwhile, it would be great to know:
- Which design aspects are not covered in the prototype?
- What core concepts do you want to materialize in the website through design? What is the idea you want to convey to the public? (Can we use the core concepts I've just share with you in my last PR?)
- Does the new prototype mean any progress in that?

**About mobile:**
I have some ideas about it. I hope to work on some wireframes tomorrow, early in the morning (power outage is approaching).

Thank you for sharing your thoughts, and your patience and support during these hard days 🌱

**Reply by davcortez** [reply]
The new design looks better.

**Reply by davcortez** [reply]
Which design aspects are not covered in the prototype? The event card component and the communities card.
The communities card should include a small logo. I don’t think it’s a good idea for it to look like the current one.

**Reply by davcortez** [reply]
What core concepts do you want to materialize in the website through design? What is the message you want to convey to the public?
Well, I thought I would share the manifesto.

**Reply by davcortez** [reply]
Does the new prototype represent any progress in that?
The homepage looks great, but the Communities and Team pages need some improvement.

**Reply by davcortez** [reply]
The Communities page should use a grid design, meaning everything should be aligned. You should avoid centering two items at the beginning and placing the others below. I’m referring to consistency— all team members should be placed next to each other, and the layout should adjust properly for smaller screens.

**Reply by nsandoya** [reply]
> The Communities page should use a grid design, meaning everything should be aligned. You should avoid centering two items at the beginning and placing the others below. I’m referring to consistency— all team members should be placed next to each other, and the layout should adjust properly for smaller screens.

Hi @davcortez 

Thank you for your feedback and recommendations 🌱

I’ve taken the mobile-first approach into account while working on the UI design, particularly for the **community landing page** and **event landing page**. 

Here are some updates:

- **Community Landing Page:** I’ve focused on ensuring that core interactions are intuitive and content is accessible on smaller screens.
- **Event Landing Page:** While I’ve implemented the mobile-first framework, I believe the header section (event info) still needs improvement. It would be great to hear your thoughts on this and explore ideas for refining it further.

Feel free to check the current progress here (you can tap on an event icon to navigate to community landing page, and vice versa): https://www.figma.com/proto/x7GTNPvqJwtyzjfKX5NqH9/ECinTech?node-id=446-2588&node-type=frame&t=jL9w1FVUKH66gKk9-1&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1

Feedback is always welcome 🙂

**Reply by nsandoya** [reply]
> Which design aspects are not covered in the prototype? The event card component and the communities card. The communities card should include a small logo. I don’t think it’s a good idea for it to look like the current one.

About this topic:

What do you think about this idea @davcortez ?: https://www.figma.com/proto/x7GTNPvqJwtyzjfKX5NqH9/ECinTech?node-id=431-13837&node-type=frame&t=79pGwQF9S9PBu09N-1&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1

**Reply by davcortez** [reply]
@nsandoya 
![Screenshot_2024-12-14_18-09-29](https://github.com/user-attachments/assets/13da0a5e-8916-4d43-aca9-6bff6c5e87a0)
I was thinking of something like this.

**Reply by nsandoya** [reply]
> @nsandoya ![Screenshot_2024-12-14_18-09-29](https://private-user-images.githubusercontent.com/104221111/395827407-13da0a5e-8916-4d43-aca9-6bff6c5e87a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ1MzMwOTksIm5iZiI6MTczNDUzMjc5OSwicGF0aCI6Ii8xMDQyMjExMTEvMzk1ODI3NDA3LTEzZGEwYTVlLTg5MTYtNGQ0My1hY2E5LTZiZmY2YzVlODdhMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxOFQxNDM5NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYWMwNGY2NTFlODUxNjk1OTYzNGVlZDcxM2FhYjU4MjYwMTEzZTlkMzg0YmU1OTA3ZjlmYzg2MjhlZTA4YTBmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Ivq3VIldMLL2mGxkINhppwHc7Lq4YDGXOcADSjoiKDU) I was thinking of something like this.

Hi @davcortez ,
Maybe we can merge the simplicity of CoDeAr reference, and some of our own style elements (drop shadow, rounded corners...). This way, we can complement communities communication efforts through design, aiming to convey better what are their core themes and values.

What do you think about these ideas? :
![Captura de pantalla 2024-12-18 110402](https://github.com/user-attachments/assets/31f8bcd9-8efd-4210-b6bd-81604012fe15)
![Captura de pantalla 2024-12-18 110140](https://github.com/user-attachments/assets/35391b86-92ea-446f-a914-40097487775e)

**Comment by davcortez** [comment]
![card](https://github.com/user-attachments/assets/eb62a86c-2c6c-4435-8cd7-08c9554ecb96)
I like this one. I would like to see the tags included to make filtering easier for users. @nsandoya

**Reply by nsandoya** [reply]
Here's a new card variant --tags included :) @davcortez 
![Captura de pantalla 2024-12-18 221107](https://github.com/user-attachments/assets/1dffe492-c755-49f7-9b29-be31ff6e20fb)

**Comment by nsandoya** [comment]
> I’m wondering how to approach the UI design for the entire site. I was looking at the Figma, but the design doesn’t cover many aspects. What about responsive design? It’s worth thinking in a mobile-first way.
> 
> @nsandoya @IvanM9

Hello @bjgavilanes @davcortez @IvanM9 ,

I hope you're doing well. Over the weekend, I worked on the website design mockups, and I wanted to share my progress with you. Due to time constraints, the mobile and desktop versions (both part of a sole responsive design) are still in separate mockups:

- [Desktop mockup](https://www.figma.com/proto/x7GTNPvqJwtyzjfKX5NqH9/ECinTech?node-id=431-13687&t=kIeqLlOv0YEqWzj6-0&scaling=scale-down-width&content-scaling=fixed&page-id=0%3A1&hide-ui=1)
- [Mobile mockup](https://www.figma.com/proto/x7GTNPvqJwtyzjfKX5NqH9/ECinTech?node-id=626-2932&t=kIeqLlOv0YEqWzj6-0&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1)

I apologize for not having them fully integrated yet, but I wanted to make sure you could review the current progress. Feedback is always welcome 🙂

Thank you!,
Nataly
